### ISSUE TITLE
Remove iframe node from <webview> will cause webview crash.

### ISSUE THREAD

**Issue by cloudsere** [electron/electron#18788]
### Preflight Checklist

* [x] I have read the [Contributing Guidelines](https://github.com/electron/electron/blob/master/CONTRIBUTING.md) for this project.
* [x] I agree to follow the [Code of Conduct](https://github.com/electron/electron/blob/master/CODE_OF_CONDUCT.md) that this project adheres to.
* [x] I have searched the issue tracker for an issue that matches the one I want to file, without success.

### Issue Details

* **Electron Version:**
  *  v5.0.2
* **Operating System:**
  * macOS 10.13.6
* **Last Known Working Electron version:**
  * Haven't tried on other electron versions.

### Expected Behavior
I have an app which runs in electron `<webview>`. I need to render multiple iframes inside this `<webview>` and dynamic add or remove them. 

### Actual Behavior
When I remove one `<iframe>` from the dom tree, the whole `<webview>` crashes. I didn't use JS to remove iframe node, instead, I just delete one `<iframe>` from the dom inspector, and the `<webview>` crashes.

### To Reproduce
This is an example : https://github.com/cloudsere/electron-quick-start. You can run commands below to reproduce.

```sh
$ git clone https://github.com/cloudsere/electron-quick-start
$ npm install
$ npm start
```

**Comment by alexstrat** [comment]
[Here](https://gist.github.com/alexstrat/eab262cdb59d1303e6b1bdce82295b4c) is a gist to be used in electron-fiddle to be used to reproduce the issue:

![](http://g.recordit.co/OcCxNZoK5n.gif)

Note that it does not seem that the `webview` itself has crashed (I can access DOM via DevTools) but rather that the repainting is getting blocked.

Tested with other versions:
- 5.0.5: reproduced
- 5.0.4: reproduced
- 4.2.4: not reproduced
- 6.0.0-beta.9: not reproduced

**Comment by alexstrat** [comment]
Relates: #17890

**Comment by alexstrat** [comment]
The fact that the bug is not reproduced with `6.0.0-beta.2` whereas it was present `beta.1` is a sign that there is a fix hiding [in the diff Chrome `76.0.3774.1` => `76.0.3783.1`](https://chromium.googlesource.com/chromium/src/+log/76.0.3774.1..76.0.3783.1?n=10000&pretty=fuller).

This one is a good candidate: https://chromium.googlesource.com/chromium/src/+/7464a6b4e3cf91dd1eb96d922736f0815b94ed9c%5E%21/#F3

**Comment by cloudsere** [comment]
> The fact that the bug is not reproduced with `6.0.0-beta.2` whereas it was present `beta.1` is a sign that there is a fix hiding [in the diff Chrome `76.0.3774.1` => `76.0.3783.1`](https://chromium.googlesource.com/chromium/src/+log/76.0.3774.1..76.0.3783.1?n=10000&pretty=fuller).
> 
> This one is a good candidate: https://chromium.googlesource.com/chromium/src/+/7464a6b4e3cf91dd1eb96d922736f0815b94ed9c%5E%21/#F3

Thank you , this fix works!

**Comment by sofianguy** [comment]
fixed https://github.com/electron/electron/pull/18976
